src/common/PropertyExt.js

```diff
@@ -129,6 +129,13 @@
             }
         }, this);
     }
+
+    PropertyExt.prototype.changeTag = function(id, tag) {
```

Comment:
rename to ```extTags``` and allow the getting as well as the setting using our usual pattern (you will need to check for args.len > 1 though.